src/Employer/Employer.Web/ViewModels/Dashboard/DashboardViewModel.cs

```diff
@@ -18,6 +18,9 @@ public class DashboardViewModel
         public int VacancyCountDraft => Vacancies.Count(v => v.Status == VacancyStatus.Draft);
         public string VacancyTextDraft => "advert".ToQuantity(VacancyCountDraft, ShowQuantityAs.None);
         public bool HasDraftVacancy => VacancyCountDraft > 0;
+        public int VacancyCountReview => Vacancies.Count(v => v.Status == VacancyStatus.Review);
+        public string VacancyTextReview => "advert".ToQuantity(VacancyCountReview, ShowQuantityAs.None);
```

Comment:
could we user the VacancyTerm constant implemented in Pauls branch?

Reply:
Paul already added the changes to the employer side https://github.com/SkillsFundingAgency/das-recruit/pull/958/files
do you want me to bring those changes (cherry pick) to this branch